docs/diagrams/SearchCommandClassDiagram.puml

```diff
@@ -0,0 +1,61 @@
+@startuml
```

Comment:
The class diagram well designed and completed, but it can be further simplified. Some basic methods like getters can be omitted, so the diagram is clearer. 👍